processing/src/main/java/io/druid/query/search/search/DefaultSearchQueryMetrics.java

```diff
@@ -0,0 +1,238 @@
+/*
+ * Licensed to Metamarkets Group Inc. (Metamarkets) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. Metamarkets licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.druid.query.search.search;
+
+import com.metamx.emitter.service.ServiceEmitter;
+import io.druid.collections.bitmap.BitmapFactory;
+import io.druid.query.BitmapResultFactory;
+import io.druid.query.Query;
+import io.druid.query.QueryMetrics;
+import io.druid.query.filter.Filter;
+import org.joda.time.Interval;
+
+import java.util.List;
+
+public class DefaultSearchQueryMetrics implements SearchQueryMetrics
```

Comment:
Please add Javadoc to this class like: `This class is implemented with delegation to another QueryMetrics for compatibility, see "Making subinterfaces of QueryMetrics for emitting custom dimensions and/or metrics for specific query types" section in {@link QueryMetrics} javadoc`